cmd/mount.go

```diff
@@ -75,22 +75,21 @@ type mountOptions struct {
 	inputMountPath string
 	ConfigFile     string
 
-	Logging            LogOptions     `config:"logging"`
-	Components         []string       `config:"components"`
-	Foreground         bool           `config:"foreground"`
-	NonEmpty           bool           `config:"nonempty"`
-	DefaultWorkingDir  string         `config:"default-working-dir"`
-	CPUProfile         string         `config:"cpu-profile"`
-	MemProfile         string         `config:"mem-profile"`
-	PassPhrase         string         `config:"passphrase"`
-	SecureConfig       bool           `config:"secure-config"`
-	DynamicProfiler    bool           `config:"dynamic-profile"`
-	ProfilerPort       int            `config:"profiler-port"`
-	ProfilerIP         string         `config:"profiler-ip"`
-	MonitorOpt         monitorOptions `config:"health_monitor"`
-	WaitForMount       time.Duration  `config:"wait-for-mount"`
-	LazyWrite          bool           `config:"lazy-write"`
-	disableKernelCache bool           `config:"disable-kernel-cache"`
```

Comment:
This flag got removed here

Reply:
this was not used, rather the flag is coming from libfuse options